### DISCUSSION TITLE
Running tests on calva using nbb repl

### DISCUSSION THREAD

**Discussion by omariosouto** [babashka/nbb#227] [Show and tell]
Here is how I'm running my tests on REPL using NBB

- Cmd + Shift + P
- Run Calva
- Select the project path (previously I open the folder root in VSCode)
- Select nbb
- I run `Calva Load/Evaluate current file and it's dependencies`
- Them I go before start the parenthesis in `t/run-tests` and everything runs and gives me the output

- Code: https://github.com/devsoutinho/labs/blob/main/_servers/webproxy/src/webproxy/modules/auth/auth_test.cljs

- Flow in GIF:
![Jun-25-2022 18-44-10](https://user-images.githubusercontent.com/13791385/175791318-e1672ea0-31bd-455a-abd0-ce788dd97583.gif)

**Comment by omariosouto** [comment]
Hey, @borkdude do you know how to run only using `Calva: Run current test`?

![run-current-test](https://user-images.githubusercontent.com/13791385/175791452-481d692d-3b24-4419-8f1f-8015e95437f9.gif)

**Reply by borkdude** [reply]
I think this is a question for @PEZ - I don't know how that's implemented in Calva.
